handout/advanced-components/view_encapsulation.md

```diff
@@ -1,13 +1,11 @@
 # View Encapsulation
 
-Defines whether the template and styles defined within the component can affect the whole application or vice versa. Angular provides three encapsulation strategies:
+Encapsulation defines whether the template and styles defined within the component can affect the whole application or vice versa. Angular provides three encapsulation strategies:
```

Comment:
We should probably say "View encapsulation defines ..."

The word "encapsulation" by itself is commonly understood by programmers to have a more general meaning that what we're talking about here.